test/evergreen.yml

```diff
@@ -1504,6 +1504,16 @@ tasks:
         vars:
           test_name: wt8246_compact_rts_data_correctness
 
+  - name: csuite-wt8659-reconstruct-database-from-logs-test
```

Comment:
Should we add a stressful test? With a higher number of iterations? It could be done in another ticket as well.

Reply:
I'm not sure, it's not my call...